src-frontend/app/templates/components/page-list.hbs

```diff
@@ -1,7 +1,7 @@
 <ul class="procedure">
     <li><a class="add-page" {{action 'addPage' 0}}>+</a></li>
     {{#each page in pages}}
-        <li class="page-template">
+        <li class="page-template" id="{{page.id}}">
```

Comment:
should probably use `data-id` here, since we're not using `id` for CSS/jQuery selectors